straxen/plugins/acqmon_processing.py

```diff
@@ -130,89 +220,145 @@ def channel_select_(rr, ch):
 
 
 @export
-@strax.takes_config(
-    strax.Option('veto_proximity_window', default=int(5e8), type=int, track=True,
-                 help='Maximum separation between veto stop and start pulses [ns]'))
 class VetoProximity(strax.OverlapWindowPlugin):
     """
-    Find the closest next/previous veto start and end to each event center.
-
-    previous_x: Time in ns between the time center of an event and the previous x
-    next_x: Time in ns between the time center of an event and the next x
-    This also considers any x inside the event. x could be either:
-        - busy_x: busy on/off signal
-        - he_x:   high energy channels busy on/off signal
-        - hev_x:  high energy veto on/off signal
+    Find the closest next/previous veto start w.r.t. the event time or
+    when a busy happens during an event.
     """
 
-    __version__ = '0.1.2'
-    depends_on = ('event_basics', 'aqmon_hits')
-    provides = ('veto_proximity')
-    data_kind = ('events')
-    save_when = strax.SaveWhen.TARGET
-    veto_names = ['busy', 'he', 'hev']
+    __version__ = '2.0.0'
+    # Strictly speaking, we could depend on 'events', but then you couldn't
+    # change the event_window_fields to e.g. s1_time and s2_endtime.
+    depends_on = ('event_basics', 'veto_intervals')
+    provides = 'veto_proximity'
+    data_kind = 'events'
+    save_when = strax.SaveWhen.NEVER
+
+    event_window_fields = straxen.URLConfig(
+        default=('time', 'endtime'),
+        help='Fields to determine where to look for overlaps for using '
+             'this plugin in the events. The default uses start and endtime '
+             'of an event, but this can also be the S1 or S2 start/endtime'
+    )
+
+    veto_proximity_window = straxen.URLConfig(
+        default=int(1e9),
```

Comment:
I think before it was `10e9`.

Reply:
yeah it should be even bigger I found out actually. I think the longest busys are order minutes